Automattic/jetpack
# Optimize Social-Logos CSS

This pull request resolves the issue I reported in #13201 

This pull request does a few things.
1. It removes the duplicate @font-face rule (to avoid the .eot) version of the file from being downloaded on browsers that don't need/use it. 
2. It replaces the data-attribute .woff file with a URL to its source instead. 
3. It includes the .woff2 file that has been bundled with the plugin for a while now to further reduce the weight. 

Fixes #13201 

(I also had a typo in one of the commits where I included woff2 twice so the additional commits were to resolve this). 

Testing Instructions
1. Activate the share buttons functionality within jetpack. 
2. Add share buttons to frontend. 
3. Verify buttons are working as expected with necessary browsers. 
4. Profit! 

#### Proposed changelog entry for your changes:

* Optimized social-logos CSS.
